README.md

```diff
@@ -1,189 +1,249 @@
-# c2paTool
+# c2patool - C2PA command line tool
 
-Command line tool for displaying and adding C2PA manifests  
-A file path to a JPEG or a claim definition JSON file must be provided  
-If a JPEG path is given, this will generate a summary report of any manifests in that file  
-If a manifest definition JSON file is specified, the manifest will be created and displayed in a JSON report
+c2patool is a command line tool for working with C2PA [manifests](https://c2pa.org/specifications/specifications/1.0/specs/C2PA_Specification.html#_manifests). Currently, the tool supports:
 
-## Displaying Manifest data
+- Reading a JSON report of C2PA manifests in an JPEG or PNG image
+- Reading a low-level report of C2PA manifest data in a JPEG or PNG image
+- Previewing manifest data from a JSON config file
+- Adding a C2PA manifest to an existing JPEG or PNG image
+- Writing a C2PA manifest to a `.c2pa` manifest file
 
-Invoking the tool with a path to an image file will output a JSON report of the Manifests in the file
-File formats supported are jpeg and png. 
+## Installation
 
-```c2patool image.jpg```
+### via Homebrew
 
-## Displaying detailed Manifest data
+You can install c2patool on Mac and Linux via [Homebrew](https://brew.sh/):
 
-The -d option will output a detailed JSON report of the internal C2PA structure
+```shell
+brew tap contentauth/tools
+brew install c2patool
+```
 
-```c2patool image.jpg  -d```
+### Building from source
 
-## Previewing a Manifest
+If you have Rust installed, you can build c2patool from source:
 
-If a path to a json config file is given,
-the tool will generate a new manifest using the values given in definition
-this will display the results but not save anything unless an output (-o) is specified
+```shell
+git clone git@github.com:contentauth/c2patool.git
+cargo build
+```
 
-```c2patool sample/config.json```
+## Usage
 
-The config json can also be passed on the command line as string using the -c --config option
+### Displaying manifest data
+
+Invoking the tool with a path to an image file will output a JSON report of the manifests contained in the file. Supported file formats are `jpeg` and `png`. 
```

Comment:
probably want to state what the supported formats are in one place and then just reference "a supported format"